### ISSUE TITLE
create shell scripts to run the BareBones and RateDelivery desks outside of Eclipse

### ISSUE THREAD

**Issue by tsember** [tsember/freight-portal#38]
I followed the steps [described here](http://stackoverflow.com/questions/574594/how-can-i-create-an-executable-jar-with-dependencies-using-maven)

**Comment by jpsember** [comment]
To compile the jar file, from Terminal, type:
```
mvn clean compile assembly:single
```
and to run it,
```
java -jar target/freight-portal-1.0-jar-with-dependencies.jar 
```

**Comment by jpsember** [comment]
Now running the program (which builds it as well) is done by typing:
```
rake
```

**Comment by jpsember** [comment]
Now solved by #65
